### DISCUSSION TITLE
Adoption

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/UniversalStorageStockResourceFuelCell#3] [Announcements]
Greetings @QuickBASIC (Mike Justman)!

I am asking for your permission to update / continue / adopt / release the below listed and named wonderful mod(s); unless somebody did already.

I absolutely love these mods and really want others to enjoy them as well.

Hopefully I hear back from you with an answer, but in case I don't I intend to respectfully proceed as soon as the middle of September 2021, as permitted by the licenses of these mods. I will provide recognition of original authors/maintainers; and I will gladly step aside should you ever want to continue this mod yourself.

Couple of requests:

I am asking for a Github post giving me permission along with tagging [ @DasSkelett ](https://forum.kerbalspaceprogram.com/index.php?/profile/188118-dasskelett/) and [ @HebaruSan ](https://forum.kerbalspaceprogram.com/index.php?/profile/156657-hebarusan/) so a smooth transition on CKAN which will allow update / release without confusing end users by keeping the CKAN listing intact. Links are included.

Any unreleased updates would be very helpful as well.

Thank you for your time and consideration!
------
### UniversalStorageStockResourceFuelCell
- MIT [![License MIT](https://camo.githubusercontent.com/4a61ff95f939dd7e623d70f71dede02c0475b5a0510d7f7359b66502f8136360/68747470733a2f2f696d672e736869656c64732e696f2f656e64706f696e743f75726c3d68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f7a6572304b657262616c2f466f756e646174696f6e732f6d61737465722f6a736f6e2f6c6963656e73652e6a736f6e)](https://camo.githubusercontent.com/4a61ff95f939dd7e623d70f71dede02c0475b5a0510d7f7359b66502f8136360/68747470733a2f2f696d672e736869656c64732e696f2f656e64706f696e743f75726c3d68747470733a2f2f7261772e67697468756275736572636f6e74656e742e636f6d2f7a6572304b657262616c2f466f756e646174696f6e732f6d61737465722f6a736f6e2f6c6963656e73652e6a736f6e) [![MIT](https://camo.githubusercontent.com/c885d1bf837794faf1272ee5f37a9ebc4ec4492a5790ddc30bd92d82c86914bc/68747470733a2f2f692e706f7374696d672e63632f62766a66734d50352f4d49542d31377831372e706e67)](https://camo.githubusercontent.com/c885d1bf837794faf1272ee5f37a9ebc4ec4492a5790ddc30bd92d82c86914bc/68747470733a2f2f692e706f7374696d672e63632f62766a66734d50352f4d49542d31377831372e706e67)
- https://github.com/QuickBASIC/UniversalStorageStockResourceFuelCell
- CKAN: UniversalStorageStockResourceFuelCell
-----

**Comment by QuickBASIC** [comment]
Yes. Thank you for offering to continue to maintain these mods.  I currently have very limited time to play KSP and therefore don't have time to continue to maintain these.
